### DISCUSSION TITLE
Evaluation of overlapping peaks

### DISCUSSION THREAD

**Discussion by trinevd** [GreenleafLab/ArchR#1098] [Feature Requests]
Hi.

Would it be possible to add info in the ArchR_Proj@peakSet$elementMetadata on which specific cluster(s) a given peak was called in originally? E.g. A peak in the uniform PeakSet may "arise" from 3 slightly shifted, but overlapping peaks, that were called in three individual clusters and later merged into one peak in the iterative merging approach.

I would like to use this info to check for overlaps of called peaks across clusters (see picture) by generation of a  Venn diagram.

Best,
Trine

![image](https://user-images.githubusercontent.com/62384776/138481729-073df755-f497-4fff-bbc3-e914d59330fb.png)

**Comment by rcorces** [comment]
A few notes on this:
1) Peak calls are noisey so just because a peak isnt called in a specific cluster doesnt mean that there isnt an enrichment in signal there. I tend to avoid making conclusions based solely on peak calls
2) You can see all of the group (aka cluster) specific peak calls which are stored in the `PeakCalls` folder. You are welcome to use standard GenomicRanges operations to find overlaps.
